services/virtual-assistant/src/virtual-assistant/run.py

```diff
@@ -19,7 +27,21 @@ async def handle_request_validation_error(error):
     return ValidationError(message=str(error.validation_error)), 400
 
 
-QuartSchema(api, openapi_path=base_url + "/openapi.json")
+def configure(binder: injector.Binder) -> None:
+    # Read configuration and assemble our dependencies
+
+    # This gets injected into routes when it is requested.
+    # e.g. async def status(session_storage: injector.Inject[SessionStorage]) -> TalkResponse:
+    if app.session_storage == "redis":
+        binder.bind(SessionStorage, to=RedisSessionStorage())
+
+    if app.session_storage == "file":
+        binder.bind(SessionStorage, to=FileSessionStorage(".va-session-storage"))
```

Comment:
It's the same, but it feels grouped and that makes sense since it is sort of a switch/case sort of stuff. It is easier to read.

```suggestion
    if app.session_storage == "redis":
        binder.bind(SessionStorage, to=RedisSessionStorage())
    elif app.session_storage == "file":
        binder.bind(SessionStorage, to=FileSessionStorage(".va-session-storage"))
```